sdk/azcore/CHANGELOG.md

```diff
@@ -1,11 +1,23 @@
 # Release History
 
-## 1.17.1 (Unreleased)
+## 1.18.0 (Unreleased)
```

Comment:
We should release this as `1.18.0-beta.1` so we have a chance to fix things before it GAs.

Reply:
You'll also need to update the version `const` [here](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/internal/shared/constants.go#L43).

Reply:
updated!